lib/repl.js

```diff
@@ -571,7 +567,8 @@ REPLServer.prototype.complete = function(line, callback) {
             this.context.constructor &&
             this.context.constructor.name === 'Context') {
           var contextProto = this.context;
-          while (contextProto = Object.getPrototypeOf(contextProto)) {
+          while (
+            (contextProto = Object.getPrototypeOf(contextProto)) === true) {
```

Comment:
IMO, we should use condition in conditional sentences.
I am happy if linter checks the following bug.

```javascript
if (a = 'test') { // a == 'test' is really needed code 
  // ...
}
```